<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeadDialog</class>
 <widget class="QDialog" name="HeadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>更换头像</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>点击更换头像</string>
   </property>
  </widget>
  <widget class="QToolButton" name="headBtn_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/桌面/pic/1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="headBtn_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/桌面/pic/2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="headBtn_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/桌面/pic/3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="headBtn_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/桌面/pic/4.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="headBtn_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/桌面/pic/5.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认更换并返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
